### PR TITLE
feat(service): create postcard service

### PR DESCRIPTION
**Pull request by nathankhuu**
What: postcard service to make HTTP call to API
Why: first step to create a postcard
Details:
Created and registered postcard service
Added create function

### CODE REVIEW

File: test/services/postcard.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
+var Config = require('../../config');
```
Comment (jesscxu) [inline test/services/postcard.spec.js]: nit: new line above (we want to separate imports from other services and files within our application)

File: test/services/postcard.spec.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect  = require('chai').expect;
```
Comment (jesscxu) [inline test/services/postcard.spec.js]: nit: alphabetize

File: app/services/index.js [hunk 3]
```diff
@@ -5,12 +5,14 @@ var Cookies = require('angular-cookies');
 
 var API     = require('./api');
 var Auth    = require('./auth');
+var Postcard = require('./postcard');
```
Comment (jesscxu) [inline app/services/index.js]: nit: align equals

**Review by jesscxu — APPROVED** [review]
LGTM!

**Review by pallavi — APPROVED** [review]
LGTM!

**Pull request merged**